src/main/java/de/hysky/skyblocker/skyblock/chat/ChatRulesConfigListWidget.java

```diff
@@ -45,7 +45,7 @@ protected int getScrollbarX() {
 
     protected void addRuleAfterSelected() {
         hasChanged = true;
-        int newIndex = children().indexOf(getSelectedOrNull()) + 1;
+        int newIndex = Math.max(children().indexOf(getSelectedOrNull()), 0);
 
         ChatRulesHandler.chatRuleList.add(newIndex, new ChatRule());
         children().add(newIndex + 1, new ChatRuleConfigEntry(newIndex));
```

Comment:
These indices passed into these two lists are different. Is this intended?

Reply:
Yes. The  children list has an entry at index 0 for the labels of the coulombs so needs to be offset by 1 
![image](https://github.com/SkyblockerMod/Skyblocker/assets/70377847/4389072f-a043-4a17-bb5c-d4f70e4fd6a1)